Publish notebook for presentation

jswan5 opened the pull request:
Notebook is good to go. Want to publish to `main` now, then work on 'Open in Callysto' button to add that functionality for the attendees.

Had to branch off `mary-update` to leave her chatbot changes there

jswan5 committed:
Test change

mgrant46 committed:
initial commit

getting started

jswan5 committed:
Remove banner

Remove banner now that notebook is no longer a slideshow

mgrant46 committed:
test cell

testing my process

mgrant46 committed:
cell change

deleted the cell

jswan5 committed:
Test again

jswan5 committed:
Modify opening text

Modify opening text to remove sections covered in presentation. Start to modify coding chunks

jswan5 committed:
Change wording

Modify wording, add and remove sections to keep focus of notebook on ethics

jswan5 committed:
Update notebook

Changed more wording, added images

jswan5 committed:
Update links

jswan5 committed:
Remove chatbot file

Leave file on `mary-update` branch, update only AI notebook for presentation

misterhay reviewed (approved):
Looks great, approved.

Some of the cells don't output anything, it might be more clear to the user that something has happened if you combine them with a subsequent cell that outputs something, or add a line like `print('Data imported')` or `print('Visualization function defined')`.

Do you want to leave the filename with a dash at the end to make sure existing links work, or change it to `ai-for-educators.ipynb`?

jswan5 committed:
Update AI notebook

Add button, update links with new filename. Add outputs to cells to make notebook easier to use for attendees

jswan5 commented:
Thanks for the comments @misterhay, I made the changes in the notebook. I also change the filename, thanks for catching that.

Not sure if you wanted to check it out again, but I updated the links as well to reflect the new filename.
